src/openforms/js/components/admin/modals/FormModal.js

```diff
@@ -10,12 +10,14 @@ const FormModal = ({
   onFormSubmit,
   children,
   extraModifiers = [],
+  ...props
```

Comment:
How does this work with the defined PropTypes?
Any undefined proptype behaves like an optional `PropTypes.any`?

Reply:
yeah I think it just does nothing about it, so won't complain or validate anything.